src/app/test/delete-modal/delete-modal.component.html

```diff
@@ -11,7 +11,7 @@ <h5 class="modal-title" id="cloneModal">Are you sure you want to delete the foll
     </ul>
   </div>
   <div class="modal-footer">
-    <button class="btn btn-info" id="confirmDeletion" (click)="deleteReports(); modal.dismiss()">Yes</button>
+    <button data-cy-delete-modal-function="confirmDeletion" class="btn btn-info" id="confirmDeletion" (click)="deleteReports(); modal.dismiss()">Yes</button>
```

Comment:
function mag je hier weglaten en deletion ook aangezien er al delete-modal staat dus: 
```suggestion
    <button data-cy-delete-modal-="confirm" class="btn btn-info" id="confirmDeletion" (click)="deleteReports(); modal.dismiss()">Yes</button>
```